test/Rings/mpoly.jl

```diff
@@ -546,4 +546,8 @@ end
   @test [f1] == gens(I)
   I2 = flag_pluecker_ideal(GF(3),[1,2,3], 4);
   @test length(gens(I2)) == 10
+
+  R, _ = polynomial_ring(QQ, 6)
+  I = flag_pluecker_ideal(R,dimension_vector, ambient_dimension, minimal=false)
```

Comment:
is there a very small example where the GB computation is quick? since your not actually testing the minimal flag here